client/src/main/java/client/PlayerInput.java

```diff
@@ -0,0 +1,38 @@
+package client;
+
+import engine.player.command.PlayerCommandType;
+
+import java.awt.event.KeyAdapter;
+import java.awt.event.KeyEvent;
+import java.util.Collection;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+
+class PlayerInput extends KeyAdapter {
+
+    private final BlockingQueue<PlayerCommandType.Type> playerCommands;
+
+    PlayerInput() {
+        this.playerCommands = new LinkedBlockingQueue<>();
+    }
+
+    void drainsTo(Collection<PlayerCommandType.Type> collectionForDrain) {
```

Comment:
what is this for? Lets buffer player commands and then get then without any drain.